config/routes.rb

```diff
@@ -1,13 +1,9 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  resource :user, except: :index
```

Comment:
This wont work. I don't have access to the user id on the frontend. I have id's for everything except user id because it is encrypted. `resources` will add id params to show, update & destroy. I just send the token back and it's up to authorize_request to set current_user for those actions.